### PR TITLE
Windowmasker/convert

### LINKED ISSUE #1792: new module: windowmasker/convert
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by alxndrdiaz**
## PR checklist

Closes #1792 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: tests/modules/nf-core/windowmasker/convert/test.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+- name: windowmasker convert test_windowmasker_convert
+  command: nextflow run ./tests/modules/nf-core/windowmasker/convert -entry test_windowmasker_convert -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/windowmasker/convert/nextflow.config
+  tags:
+    - windowmasker/convert
+    - windowmasker
+  files:
```
Comment (DLBPointon) [inline tests/modules/nf-core/windowmasker/convert/test.yml:6]: As your module has the potential to output 3 (4?) different formats, you'll need a test per output.

See, this as an example:
https://github.com/nf-core/modules/blob/master/tests/modules/nf-core/bamtools/convert/test.yml

File: tests/modules/nf-core/windowmasker/convert/nextflow.config [hunk 2]
```diff
@@ -0,0 +1,9 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
```
Comment (DLBPointon) [inline tests/modules/nf-core/windowmasker/convert/nextflow.config:4]: Test per output again, see:

https://github.com/nf-core/modules/tree/master/tests/modules/nf-core/bamtools/convert

**Comment by DLBPointon** [comment]
Hi [alxndrdiaz](https://github.com/alxndrdiaz), 
Sorry for taking a while to get around to this, GitHub notifications are filling up my inbox like no one's business. I've added a few notes above that you'll need to have a look at.

**Comment by alxndrdiaz** [comment]
> Hi [alxndrdiaz](https://github.com/alxndrdiaz), Sorry for taking a while to get around to this, GitHub notifications are filling up my inbox like no one's business. I've added a few notes above that you'll need to have a look at.

Hi [@DLBPointon](https://github.com/DLBPointon) . Thanks for the review. I already pushed the changes .

**Comment by alxndrdiaz** [comment]
Commit [Merge branch 'master' into windowmasker/convert](https://github.com/nf-core/modules/pull/3152/commits/02c938a4db57259ef36f2205fdfe2b6085b2f6f8) is messing with the linting, I still need to figure out how to fix it.

**Review by muffato — APPROVED** [review]

**Comment by muffato** [comment]
Good, thank you !

**Pull request merged**